src/features/list-view/styles/MetadataListViewMessage.scss

```diff
@@ -9,8 +9,8 @@
     }
 
     .message-header-text {
-        font-weight: bold;
         font-size: 10px;
+        font-weight: bold;
```

Comment:
Unrelated change?

Reply:
this is from linting fix, not my change, but probably we should check it in